server/controllers/gym/postImage.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Response } from 'express';
+import Image from '../../database/models/images';
+import { gymImageSchema, CustomError } from '../../utils';
+import { uploadImage } from '../../utils/aws';
+
+export default async function postImage(req: any, res: Response, next: NextFunction) {
+  try {
+    const { images } = await gymImageSchema.validateAsync(req.body);
+    const { id } = req.token;
+
+    if (!images) {
+      throw new CustomError('عذرا اعد إضافة الصور ! ', 409);
+    }
```

Comment:
No need for this condition, If there's an error in the validation it will go directly to the `catch` block